packages/remark-mdx/test/fixtures/block-parsing.out

```diff
@@ -4,9 +4,9 @@ From [GitHub](https://github.com/mdx-js/mdx/issues/195#issuecomment-506745859).
 
 <Row>
   <Col>
-    -   some md
-    -   asd
-    -   asd
+    *   some md
+    *   asd
+    *   asd
```

Comment:
`remark-stringify` has some new output defaults, which are producing some noise here but are otherwise syntactically the same